backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/network/host/HostSetupNetworksCommand.java

```diff
@@ -340,26 +340,43 @@ protected void executeCommand() {
 
         try (EngineLock monitoringLock = acquireMonitorLock("Host setup networks")) {
             int timeout = getSetupNetworksTimeout();
-            FutureVDSCall<VDSReturnValue> setupNetworksTask = invokeSetupNetworksCommand(timeout);
 
             try {
-                VDSReturnValue retVal = setupNetworksTask.get(timeout, TimeUnit.SECONDS);
-                if (retVal != null) {
-                    if (!retVal.getSucceeded() && retVal.getVdsError() == null && getParameters().rollbackOnFailure()) {
-                        throw new EngineException(EngineError.SETUP_NETWORKS_ROLLBACK, retVal.getExceptionString());
+                boolean networksUpdated = false;
+                boolean succeeded = false;
+
+                if (hasNetworkChanges()) {
+                    //Update networks configuration on the host
```

Comment:
Please add space after `//` and period at the end, that applies to all comments.

Reply:
Fixed